src/status_im2/contexts/quo_preview/list_items/user_list.cljs

```diff
@@ -50,24 +49,12 @@
                              :untrustworthy? false
                              :online?        false})]
     (fn []
-      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+      [preview/preview-container
+       {:state      state
+        :descriptor descriptor}
        [rn/view {:padding-bottom 150}
-        [rn/view {:flex 1}
-         [preview/customizer state descriptor]]
         [rn/view
          {:padding-vertical    60
           :padding--horizontal 15
           :justify-content     :center}
```

Comment:
Since you're migrating all components, it would be better to remove all unnecessary code as well

```clojure
[preview/preview-container
       {:state      state
        :descriptor descriptor}
       [user-list/user-list @state]]
```
Simply passing the component directly to the `preview-container` is sufficient, there's no need for additional wrappers.

Reply:
Okay, maybe we can create a dedicated PR for a full cleanup of all preview screens. Probably after making all suggested changes to the preview setup.